Parsers/ASimAuthentication/ARM/README.md

```diff
@@ -9,6 +9,8 @@ This template deploys the following:
 * vimAuthenticationAWSCloudTrail
 * vimAuthenticationOktaSSO
 * vimAuthenticationWindowsSecurityEvent
+* vimAuthenticationMicrosoftWindowsEvent
```

Comment:
This file changes since you cloned, just remove your changes and will we add it afterwards (for Readmes I just update online as an admin).